Tests/UI.Tests/UnitTests/ControllerTests/PlayedGameControllerTests/SearchHttpPostTests.cs

```diff
@@ -50,7 +50,7 @@ public void SetUp()
         [Test]
         public void ItReturnsTheCorrectView()
         {
-            AutoMocker.Get<IPlayedGameRetriever>().Expect(mock => mock.SearchPlayedGames(Arg<PlayedGameFilter>.Is.Anything)).Return(new List<PlayedGameSearchResult>());
+            AutoMocker.Get<IPlayedGameRetriever>().Expect(mock => mock.SearchPlayedGames(Arg<PlayedGameFilter>.Is.Anything, CurrentUser)).Return(new List<PlayedGameSearchResult>());
```

Comment:
I'm using `CurrentUser` because it was already available. Is that fine or would you prefer `null`?